doc/articles/features/Lottie.md

```diff
@@ -9,16 +9,17 @@ uid: Uno.Features.Lottie
 
 * The `Microsoft.Toolkit.Uwp.UI.Lottie` namespace provides classes for rendering Lottie animations in a `Microsoft.UI.Xaml.Controls.AnimatedVisualPlayer`.
 
-## Using the `LottieVisualSource`:
+## Using the `LottieVisualSource`
 
-# [**WinUI 3**](#tab/winui)
+### [**WinUI 3**](#tab/winui)
```

Comment:
@Youssef1313 adjusting the header order does not seem to impact visually the tabs with docfx. So in my opinion MD025 should not be disabled globally I think
- PR changes: ![image](https://github.com/unoplatform/uno/assets/16295702/1354ae3d-47f4-4d17-b210-07df5a503b83)
- Prod: ![image](https://github.com/unoplatform/uno/assets/16295702/1dedd19e-f078-4a2a-afdd-f273d7f3314e)

Reply:
Good to know! I didn't know tabs works with any heading level.